release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -3157,6 +3157,8 @@ In the following tables, features are marked with the following statuses:
 
 * When installing a cluster on {aws-first} using Local Zones, edge nodes fail to deploy if deployed in the `us-east-1-iah-2a` region. (link:https://issues.redhat.com/browse/OCPBUGS-35538[*OCPBUGS-35538*])
 
+* Installing {product-title} {product-version} with the Infrastructure Operator, Central Infrastructure Management, or ZTP methods using ACM versions 2.10.3 or earlier is not possible. This is because of a change in the dynamically linked installer binary,`openshift-baremetal-install` which in {product-title} {product-version} requires a Red Hat Enterprise Linux (RHEL) 9 host to run successfully. Future versions of ACM will use the statically linked binary to avoid this issue. (link:https://issues.redhat.com/browse/MGMT-18015[*MGMT-18015*])
```

Comment:
Typically we do not like to add future forward [promises](https://redhat-documentation.github.io/supplementary-style-guide/#statements-about-the-future:~:text=it%20is%20not.-,Future%20releases%20or%20plans,-When%20possible%2C%20avoid) or predictions in our documentation. Maybe we can write this though to work. Perhaps, anticipate or plan can be used to help us here.